src/bentoml/__init__.py

```diff
@@ -154,7 +155,7 @@
     exceptions = _LazyLoader("bentoml.exceptions", globals(), "bentoml.exceptions")
     monitoring = _LazyLoader("bentoml.monitoring", globals(), "bentoml.monitoring")
     cloud = _LazyLoader("bentoml.cloud", globals(), "bentoml.cloud")
```

Comment:
Should probably deprecate this module.

Reply:
We also have push/pull bentos and models methods within the cloud client, I think we might need to continue supporting them? We can probably just remove `deployment` client from our `BentoCloudClient`?

Reply:
we should deprecate the deployment from cloud then.